honeybadgermpc/hbavss_batch.py

```diff
@@ -0,0 +1,235 @@
+import logging
+import asyncio
+from pickle import dumps, loads
+from honeybadgermpc.betterpairing import ZR
+from honeybadgermpc.polynomial import polynomials_over
+from honeybadgermpc.poly_commit import PolyCommit
+from honeybadgermpc.symmetric_crypto import SymmetricCrypto
+from honeybadgermpc.exceptions import HoneyBadgerMPCError
+from honeybadgermpc.protocols.reliablebroadcast import reliablebroadcast
+
+# TODO: Move these to a separate file instead of using it from batch_reconstruction.py
+from honeybadgermpc.batch_reconstruction import subscribe_recv, wrap_send
+
+
+class HbAVSSMessageType:
```

Comment:
I know that you are going to merge the two files later so remember to keep only class.

Reply:
Merged in ``hbavss.py`` with inheritance.